webapp/src/app/user/review-activity-card/review-activity-card.component.html

```diff
@@ -11,10 +11,16 @@
 } @else {
   <a hlmCard variant="profile" class="hover:bg-accent/50" [href]="htmlUrl() ?? this.pullRequest()?.htmlUrl" target="_blank" rel="noopener noreferrer" [class]="class()">
     <div hlmCardContent variant="profile">
-      <div class="text-sm text-muted-foreground first-letter:uppercase">
-        {{ relativeActivityTime() }}
-        in
-        {{ this.repositoryName() }} #{{ this.pullRequest()?.number }}
+      <div class="flex items-center justify-between">
+        <span class="text-sm text-muted-foreground first-letter:uppercase">
+          {{ relativeActivityTime() }}
+          in
+          {{ this.repositoryName() }} #{{ this.pullRequest()?.number }}
+        </span>
+        <div class="flex items-center text-violet-500 dark:text-violet-600">
+          <hlm-icon size="sm" name="lucideAward" />
```

Comment:
Does it make sense to also include this icon next to `Score` in the leaderboard in the same color? 
Maybe move the `text-violet` into the tailwind config to define a custom color for the points so we keep it consistent if we use it multiple times.

Reply:
Absolutely (commit 403bf25).
I had a look at our current color system (https://primer.style/foundations/primitives/color) and switched to `github-done-foreground`, which looks extremely similar.
Also the icon next to the score looks good:
![image](https://github.com/user-attachments/assets/75ce2df3-40e0-41a4-b865-9d78e4d7d626)